sys/log/include/log/log.h

```diff
@@ -51,8 +51,9 @@ struct log_handler {
 
 struct log_entry_hdr {
     int64_t ue_ts;
-    uint16_t ue_level;
     uint16_t ue_module;
+    uint16_t ue_index;
+    uint8_t ue_level;
```

Comment:
Fix is to have a log_level and index be a byte each. The problem is also that the alignment is going to be 8 bytes and so the sizeof the structure is going to be 16 bytes. I am going to pack it sop that it uses less space as this is going to go on the flash.